ci-helpers/gem_helper

```diff
@@ -36,7 +37,7 @@ module AtomicCacheCi
     end
 
     def git_client
-      @git_client ||= Git.open('.')
+      @git_client ||= Git.open('.', log: Logger.new(STDOUT))
```

Comment:
ruby-git doesn't have a default logger, so this should help us debug issues more easily in the future.